obs-browser-page/obs-browser-page-main.cpp

```diff
@@ -79,8 +79,10 @@ int CALLBACK WinMain(HINSTANCE, HINSTANCE, LPSTR, int)
 	std::thread shutdown_check;
 
 	CefMainArgs mainArgs(nullptr);
+#if CHROME_VERSION_BUILD < 5615
 	if (!SetHighDPIv2Scaling())
```

Comment:
Is this handled differently on Windows with versions >= 5615, making this unnecesary?

Reply:
Yes, the function is entirely removed and the behaviour is default instead.